cfme/automate/dialog_import_export.py

```diff
@@ -19,6 +19,7 @@ class DialogImportExportView(BaseLoggedInPage):
     title = Text("#explorer_title_text")
     upload_file = FileInput(id="upload_file")
     upload = Button(id="upload_service_dialog_import")
+    export = Text('//button[text()="Export" and @name="button"]')
```

Comment:
Should this be a button widget?

Reply:
I tried with button widget and different xpath but not able to access. So I used Text widget.